scripts/download_payload.json

```diff
@@ -0,0 +1,108 @@
+{
```

Comment:
is this exactly what you need from the state ?

Reply:
I think it should be, as discussed in the design ticket: https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-3115
We won't need a lot of the ephemeral bits of the state, such as selected file options etc.